app/src/main/java/com/example/android/navigation/GameWonFragment.kt

```diff
@@ -37,10 +38,36 @@ class GameWonFragment : Fragment() {
             view?.findNavController()?.navigate(GameWonFragmentDirections.actionGameWonFragmentToGameFragment())
         }
         displayArgs()
+        setHasOptionsMenu(true)
         return binding.root
     }
-    private fun displayArgs(){
+
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        super.onCreateOptionsMenu(menu, inflater)
+        inflater.inflate(R.menu.winner_menu, menu)
+        if (null == getShareIntent().resolveActivity(activity!!.packageManager)) {
+            menu.findItem(R.id.share)?.isVisible = false
+        }
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        when (item.itemId) {
+            R.id.share -> shareSuccess()
```

Comment:
How could we make to share this on fb/instagram?

Reply:
We need to use Facebook SDK